content/docs/guides/configuration.md

```diff
@@ -40,6 +40,32 @@ by changing the `defaultSchema` property to some other value. For example, to ch
 
 [Assertions](assertions) are created inside a different schema as specified by the `assertionsSchema` property.
 
+### Running operations in context
+
+BigQuery and SQL Data Warehouse by default run all operations for a file in the same context by default. This functionality is not supported for Redshift or Snowflake.
```

Comment:
This is useful for scripting, for example defining variables or UDFs in BigQuery before the create table statement.